frontend/src/components/createProjectForm.tsx

```diff
@@ -0,0 +1,282 @@
+import useTranslation from "next-translate/useTranslation";
+import { Badge, Col, Row } from "react-bootstrap";
+import styles from "../styles/createProjectForm.module.css";
+import { Field, Form, Formik } from "formik";
+import useSWR from "swr";
+import apiPaths from "../properties/apiPaths";
+import { getAllUsersFromPage } from "../api/calls/userCalls";
+import { capitalize } from "../utility/stringUtil";
+import { ProjectCreationProps, ProjectCreationValues } from "../handlers/createProjectSubmitHandler";
+import { IUser } from "../api/entities/UserEntity";
+import { getAllSkillTypesFromPage } from "../api/calls/skillTypeCalls";
+import { ISkillType } from "../api/entities/SkillTypeEntity";
+import { ChangeEvent, useState } from "react";
+import Image from "next/image";
+
+export const CreateProjectForm = (props: ProjectCreationProps) => {
+    let userResponse = useSWR(apiPaths.users, getAllUsersFromPage);
+
+    let users: IUser[] = userResponse.data || [];
+    let userError: Error = userResponse.error;
+
+    let skillTypeResponse = useSWR(apiPaths.skillTypes, getAllSkillTypesFromPage);
+
+    let skillTypes: ISkillType[] = skillTypeResponse.data || [];
+    let skillTypeError: Error = skillTypeResponse.error;
+
+    const { t } = useTranslation("common");
+
+    const [coaches, setCoaches] = useState<string[] | []>([]);
```

Comment:
I'm not sure but I think this should work as wel
```suggestion
    const [coaches, setCoaches] = useState<string[]>([]);
```